packages/ubuntu_desktop_installer/lib/pages/theme/theme_page.dart

```diff
@@ -29,30 +31,51 @@ class ThemePage extends ConsumerWidget {
       title: YaruWindowTitleBar(
         title: Text(lang.chooseYourLookPageTitle),
       ),
-      content: Center(
-        child: Row(
-          mainAxisSize: MainAxisSize.min,
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            _ThemeOptionCard(
-              width: width / 3,
-              assetName: 'assets/theme/light-theme.png',
-              selected: Theme.of(context).brightness == Brightness.light,
-              onTap: () => model.setBrightness(Brightness.light),
-              preferenceName: lang.chooseYourLookPageLightSetting,
-            ),
-            SizedBox(
-              width: width / 20,
-            ),
-            _ThemeOptionCard(
-              width: width / 3,
-              assetName: 'assets/theme/dark-theme.png',
-              selected: Theme.of(context).brightness == Brightness.dark,
-              onTap: () => model.setBrightness(Brightness.dark),
-              preferenceName: lang.chooseYourLookPageDarkSetting,
+      content: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Row(
+            mainAxisSize: MainAxisSize.min,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              _ThemeOptionCard(
+                width: width / 3,
+                assetName: 'assets/theme/light-theme.png',
+                selected: Theme.of(context).brightness == Brightness.light,
+                onTap: () => model.setBrightness(Brightness.light),
+                preferenceName: lang.chooseYourLookPageLightSetting,
+              ),
+              SizedBox(
+                width: width / 20,
+              ),
+              _ThemeOptionCard(
+                width: width / 3,
+                assetName: 'assets/theme/dark-theme.png',
+                selected: Theme.of(context).brightness == Brightness.dark,
+                onTap: () => model.setBrightness(Brightness.dark),
+                preferenceName: lang.chooseYourLookPageDarkSetting,
+              ),
+            ],
+          ),
+          if (yaru != null)
+            Center(
+              child: SingleChildScrollView(
+                padding: const EdgeInsets.only(top: kWizardSpacing * 2),
+                scrollDirection: Axis.horizontal,
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    for (final variant in YaruVariant.values.take(10))
```

Comment:
```suggestion
                    for (final variant in YaruVariant.accents)
```
- https://github.com/ubuntu/yaru.dart/pull/359